src/bin/random.rs

```diff
@@ -0,0 +1,35 @@
+extern crate bytebeat;
+extern crate rand;
+
+use rand::Rng;
+use bytebeat::random;
+mod util;
+
+fn main() {
```

Comment:
It might be nice to make the random program only generate a formula.
Then you can do something like
```sh
# Assume ./target/release/... or cargo run --bin or whatever
# Generate a nicely colored random formula video
random | python3 gencol.py '#bbrandom' | encode
# Make some audio
random | bytebeat | sox
# Tweet
random | python3 gencol.py '#bbrandom' | bot
```

Reply:
Hmm.. this does actually seem more useful then, I'll do it!